st/clitests/s3cmd_spec.py

```diff
@@ -61,6 +61,8 @@
 Config.config_file = "pathstyle.s3cfg"
 
 # ************ Test post removal of data (removal/creation of root index) ************
+# Recreating of global indices causes s3server can loose access to them.
+'''
 s3kvs.clean_all_data()
```

Comment:
As I mentioned earlier, we have s3kvs.clean_all_data() is many ST spec file. This is another place that we have this call. Why this is not commented (like in BD ST spec file)?